ts/Dashboards/EditMode/EditMode.ts

```diff
@@ -1043,14 +1043,21 @@ class EditMode {
      * Whether the edit overlay should be removed.
      */
     public setEditOverlay(
-        remove?: boolean
+        remove?: boolean,
+        forceRemove?: boolean
     ): void {
         const editMode = this,
             cnt = editMode.editOverlay,
             isSet = cnt?.classList.contains(
                 EditGlobals.classNames.editOverlayActive
             );
 
+        if (forceRemove) {
+            cnt?.classList.remove(EditGlobals.classNames.editOverlayActive);
+            editMode.isEditOverlayActive = false;
+            return;
+        }
+
```

Comment:
I am a bit confused about `forceRemove`. To me it seems to do just the same as `remove`, but without checking if overlay is set.
Why do we need both params? I really think we should manage with just `remove`, unless there is something i dont understand here

Reply:
I'm not convinced too. Its confusing.